source/_integrations/smartthings.markdown

```diff
@@ -136,22 +49,20 @@ SmartThings represents devices as a set of [capabilities](https://developer.smar
 - [Scene](#scene)
 - [Switch](#switch)
 
-Support for additional platforms will be added in the future.
-
 ### Binary sensor
 
-The SmartThings binary sensor platform lets you view devices that have binary sensor-related capabilities. A binary sensor entity will be created for each attribute (below) supported by the device.
+A binary sensor entity will be created for each of the following capabilities if the device has those.
 
-| Capability                                                                                                                    | Attribute      | On-Value   |
-| ----------------------------------------------------------------------------------------------------------------------------- | -------------- | ---------- |
-| [`accelerationSensor`](https://developer.smartthings.com/docs/devices/capabilities/capabilities-reference#accelerationSensor) | `acceleration` | `active`   |
-| [`contactSensor`](https://developer.smartthings.com/docs/devices/capabilities/capabilities-reference#contactSensor)           | `contact`      | `open`     |
-| [`filterStatus`](https://developer.smartthings.com/docs/devices/capabilities/proposed#filterStatus)                           | `filterStatus` | `replace`  |
-| [`motionSensor`](https://developer.smartthings.com/docs/devices/capabilities/capabilities-reference#motionSensor)             | `motion`       | `active`   |
-| [`presenceSensor`](https://developer.smartthings.com/docs/devices/capabilities/capabilities-reference#presenceSensor)         | `presence`     | `present`  |
-| [`tamperAlert`](https://developer.smartthings.com/docs/devices/capabilities/capabilities-reference#tamperAlert)               | `tamper`       | `detected` |
-| [`valve`](https://developer.smartthings.com/docs/devices/capabilities/capabilities-reference#valve)                           | `valve`        | `open`     |
-| [`waterSensor`](https://developer.smartthings.com/docs/devices/capabilities/capabilities-reference#waterSensor)               | `water`        | `wet`      |
+| Capability                                                                                                                    |
+|-------------------------------------------------------------------------------------------------------------------------------|
+| [`accelerationSensor`](https://developer.smartthings.com/docs/devices/capabilities/capabilities-reference#accelerationSensor) |
```

Comment:
```suggestion
| `accelerationSensor`|
```
1. Do you think it is necessary to send the user to the dev docs? Or is it enough if the capabilities are listed?
2. Is this how their capability is spelled in their front end? Is this the UI string I see in their app?